styleguide/derek/_partials/solutions/quote.ejs

```diff
@@ -1,22 +1,12 @@
 <!-- Testimonials -->
-<div class="bg-blue-gray standard-padding">
-  <div class="container">
-    <h4 class="white">Quote Split</h4>
-    <div id="quote">
-      <div class="row">
+<div class="quote">
+  <div class="quote-wrapper">
+    <div class="row">
         <div class="col-md-6">
-          <div class="row">
-            <div class="col-md-8 center-block">
-              <blockquote class="center white">Because of Fanatical Support, we knew we were partnering with a best-in-class company that shared our vision of getting it done.</blockquote>
-              <cite class="center">Brian McManus, Senior Director of Technology - Under Armour</cite>
-            </div>
-          </div>
-          <!--Option for logo-->
-          <div class="row red semi-bold half-padding-top">
-            <div class="col-md-2 col-xs-2 center-block">
-              <img src="https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/logos/underarmor.png" alt="">
-            </div>
-          </div>
+          <%- partial('../../_partials/cards/quote.ejs', {
+            textWrapperClass: 'quote-split-textWrapper',
+            logoClass: 'quote-split-logo'
+          }) %>
```

Comment:
@WidgetsBurritos so there are two different quote "patterns" the quote and the split quote. They have two different files. I may just not be super duper clear. We can chat about it in wheaties.